core/src/main/java/org/apache/accumulo/core/client/ConditionalWriterConfig.java

```diff
@@ -133,4 +137,39 @@ public int getMaxWriteThreads() {
   public Durability getDurability() {
     return durability;
   }
+
+  /**
+   * Sets the name of the classloader context on this scanner. See the administration chapter of the user manual for details on how to configure and use
+   * classloader contexts.
+   *
+   * @param classLoaderContext
+   *          name of the classloader context
+   * @throws NullPointerException
+   *           if context is null
+   * @since 1.8.0
+   */
+  public void setClassLoaderContext(String classLoaderContext) {
+    Preconditions.checkNotNull(classLoaderContext, "context name cannot be null");
+    this.classLoaderContext = classLoaderContext;
+  }
+
+  /**
+   * Clears the current classloader context set on this scanner
+   *
+   * @since 1.8.0
+   */
+  public void clearClassLoaderContext() {
```

Comment:
AFAICT this is not used in a test.  A test that scans w/ a context, clears the context, and then scans again would be nice.